ast/src/test/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitorTest.java

```diff
@@ -580,4 +580,34 @@ public void testParseAndPrintNestedLoop() throws Exception {
     )));
   }
 
+  @Test
+  public void addBracesParameterTest() throws Exception {
+    final String frag =
+          "void main(void)\n"
+        + "{\n"
+        + "  if (condition)\n"
+        + "  do_something();\n"
+        + "}\n";
+    final String expected =
+          "void main(void)\n"
+        + "{\n"
+        + "  if (condition) {\n"
+        + "  do_something();\n"
+        + "  }\n"
+        + "}\n";
+
+    final File fragFile = temporaryFolder.newFile("shader.frag");
+    final File outFile = temporaryFolder.newFile("out.frag");
+    FileUtils.writeStringToFile(fragFile, frag, StandardCharsets.UTF_8);
```

Comment:
Minor thing: could you create an instance of ShaderJobFileOperations() and use that for all file operations. We don't do this consistently, but we should with new code. If it isn't available (because it isn't a dependency of this module) then don't worry.

Reply:
ShaderJobFileOperations is not available in this scope (would cause circular dependency)